lighthouse-core/test/computed/metrics/total-blocking-time-test.js

```diff
@@ -72,27 +63,21 @@ Object {
       const fcpTimeMs = 1500;
       const interactiveTimeMs = 2500;
 
-      expect(CumulativeLongQueuingDelay.calculateSumOfLongQueuingDelay(
-        events,
-        fcpTimeMs,
-        interactiveTimeMs
-      )).toBe(150);
+      expect(
+        TotalBlockingTime.calculateSumOfBlockingTime(events, fcpTimeMs, interactiveTimeMs)
+      ).toBe(150);
     });
 
-    it('clips queuing delay regions properly', () => {
-      const fcpTimeMs = 1050;
-      const interactiveTimeMs = 2050;
+    it('treats the end of task as blocking time', () => {
+      const fcpTimeMs = 100;
+      const interactiveTimeMs = 200;
+      // The last 50ms of this 100ms task is blocking time, and it happens after FCP, so it's
+      // counted towards TBT.
+      const events = [{start: 50, end: 150, duration: 100}];
 
-      const events = [
-        {start: 1000, end: 1110, duration: 110}, // Contributes 10ms.
-        {start: 2000, end: 2200, duration: 200}, // Contributes 50ms.
```

Comment:
and under the new model this is...still impossible? ;)

Reply:
it seems like this test still exists just with different numbers, is that intentional? (the second task spans TTI)

Reply:
It was not intentional, but I just realized that TTI can happen at FMP or DCL, which can happen in the middle of a task, so this case is not as impossible as I thought. Testing this case is worth it now.